src/components/Forms/BusinessSignUpForm.js

```diff
@@ -143,105 +148,110 @@ const BusinessSignUpForm = ({ isFundraiser = false }) => {
   };
 
   //renders in place of form once it has been submitted
-  if (submitted) return <Text fontSize="2xl">Thank you for signing up!</Text>;
+  if (submitted) {
+    return <Text fontSize="2xl">Thank you for signing up!</Text>;
+  }
 
   return (
-    <FormControl
-      width="100%"
-      maxWidth="1000px"
-      margin="0 auto 3rem"
-      padding="0 24px"
-      onKeyPress={event => {
-        if (event.key === 'Enter') {
-          handleSubmit(event);
-        }
-      }}
-    >
+    <form onSubmit={handleSubmit} method="POST" noValidate>
```

Comment:
Wrapping the form in an actual `form` removes the need to listen for a specific key press. It also should provide a slightly more accessible experience.